internal/cmd/root.go

```diff
@@ -137,6 +138,14 @@ It outputs a graph representation of the build process.`,
 		"display the version of dockerfilegraph",
 	)
 
+	rootCmd.Flags().StringVarP(
+		&fileFlag,
+		"file",
+		"f",
+		"Dockerfile",
+		"name of the Dockerfile",
+	)
+
```

Comment:
Please move this up a little so that we keep the the alphabetical sort order of the flags.